docs/src/architecture.md

```diff
@@ -5,30 +5,30 @@ The architecture reflects the design goals for the rollup:
 
 * **High throughput**
 * **Privacy**
-* **Asset Safety**
+* **Asset safety**
 
 On Miden, developers can build dApps currently infeasible anywhere else.
 
-## Actor Model
-The [Actor Model](https://en.wikipedia.org/wiki/Actor_model) inspires Miden to achieve concurrent and local state changes. Actors are little state machines with inboxes, meaning each actor is responsible for their state. Actors can send and receive messages to communicate with other actors. Messages can be read asynchronously.
+## Inspired by the Actor Model
+The [Actor Model](https://en.wikipedia.org/wiki/Actor_model) inspired Miden to achieve concurrent and local state changes. In the model, actors are little state machines with inboxes, meaning each actor is responsible for their own state. Actors can send and receive messages to communicate with other actors. Messages can be read asynchronously.
```

Comment:
This might be a wider discussion. In my mind, to make the rollup docs a good read we should only explain what is necessary. To me, it is not important to show alternatives and pros and cons. 

Maybe we can have a section in every page with further information for people to read? That way, a "normal" reader what only see what is necessary to understand and more interested readers can dive deeper.